rakelib/compile.rake

```diff
@@ -2,13 +2,7 @@ require "rake/extensiontask"
 
 GEMSPEC = Bundler.load_gemspec("wasmtime-rb.gemspec")
 
-CROSS_PLATFORMS = [
-  "aarch64-linux",
-  "arm64-darwin",
-  "x86_64-darwin",
-  "x86_64-linux",
-  "x86_64-linux-musl"
-]
+CROSS_PLATFORMS = [ENV["RUBY_TARGET"]].compact
```

Comment:
If I get this right, it should've been that way from the beginning, right? We run the  compile task once per target as per the matrix, and we get the [`RUBY_TARGET` from the Dockerfiles](https://github.com/oxidize-rb/rb-sys/blob/fa0f82885d9fff747af024288636cb6e4eaf4eed/docker/setup/rubybashrc.sh#L54).

Reply:
Yup exactly. Seems like the sane default for GitHub actions, would be nice to have that be the default for RCD somehow...